### DISCUSSION TITLE
How to import TypeScript types?

### DISCUSSION THREAD

**Discussion by dccarmo** [neet/masto.js#1075] [Q&A] [answered]
Hi,

When we're fetching statuses from the timeline, for example, we would like to import the `Status` type that gets returned to be used on our project internally. Is there a way to do this?

**Comment by neet** [comment, marked as answer]
Hi @dccarmo, thank you for posting a question

Basically, all types are exported under `mastodon` namespace due to the naming convention. This is because some entity has breaking changes between different versions. For example, if you want to use `Status` type:

```ts
import type { mastodon } from "masto";

function statusHandler(status: mastodon.v1.Status) {
  //...
}
```

I'm planning improve the document in the future for easily understanding this restriction.

Hope this helps

**Reply by dccarmo** [reply]
That's perfect, thank you so much!
